content/en/logs/guide/correlate-your-full-stack-data.md

```diff
@@ -0,0 +1,210 @@
+---
+title: Correlate your full stack logs, traces and views
+kind: guide
+aliases:
+  - /logs/guide/xxx
+further_reading:
+  - link: /logs/guide/logs-rbac/
+    tag: Documentation
+    text: "Set up Roles Based Access Controls (RBAC) for Log Management"
+  - link: /agent/logs/advanced_log_collection
+    tag: Documentation
+    text: "Filter and Redact logs with Advanced Log Collection"
+  - link: /agent/guide/autodiscovery-management/
+    tag: Documentation
+    text: "Exclude containers from Log Collection with Autodiscovery"
+
+---
+
+## Overview
+
+With [unified service tagging][1] you already have high level correlation capabilities. But sometimes the starting point of your investigation is a single log or a single trace. Correlating them with other data gives context to estimate business impact and find root causes in a few clicks.
+
+{{< img src="logs/guide/correlate-your-full-stack-data/full-stack-cover.png" alt="Full stack correlation" style="width:80%;" >}}
+
+Correlating your logs also eases [aggressive sampling strategy based on Trace ID][2] without losing entity-level consistency.
```

Comment:
"based on Trace ID" should be mention for example purpose only, therefore not included in the link. Let's mention an example of a sampling strategy involving RUM as well.